src/Controller/Endpoint.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Makaira\OxidConnectEssential\Controller;
+
+use Makaira\OxidConnectEssential\HttpException;
+use Makaira\OxidConnectEssential\Rpc\RpcService;
+use Makaira\OxidConnectEssential\SymfonyContainerTrait;
+use OxidEsales\Eshop\Application\Controller\FrontendController;
+use OxidEsales\Eshop\Core\Registry;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+use function explode;
+use function ini_set;
+
+use const PHP_EOL;
+
+class Endpoint extends FrontendController
+{
+    use SymfonyContainerTrait;
+
+    public function render()
+    {
+        ini_set('html_errors', 'off');
+
+        $response = $this->handleRequest(Request::createFromGlobals());
+
+        $response->send();
+        exit(0);
```

Comment:
The `exit(0);` is required, because OXID has it's own logic after calling the `render` method. If we remove it, it will break the response.
May be a `Registry::getSession()->freeze()` should be added right before the `exit`.

Reply:
The `exit(0);` is required, because OXID has it's own logic after calling the `render` method. If we remove it, it will break the response.
May be a `Registry::getSession()->freeze()` should be added right before the `exit`.